src/mongodb/index.ts

```diff
@@ -0,0 +1,7 @@
+const uri = "mongodb+srv://ivan_kuxta:ivan_kuxta@learnnodejs.nlxd0u0.mongodb.net/?retryWrites=true&w=majority";
```

Comment:
Do not push sensitive data to github. Use .env for it.

Reply:
I wanted to use this file, but for clarity I wrote the data to a variable. I will fix it soon.